google/cloud/storage/blob.py

```diff
@@ -1877,10 +1876,10 @@ def _initiate_resumable_upload(
         :type num_retries: int
         :param num_retries:
             Number of upload retries. By default, only uploads with
```

Comment:
This docstring is kind of misleading because the default behavior for this method is no retry, but I guess it's not really worth messing with here.

Reply:
Yep that is more for backwards-compatibility. This is a private method that is supposedly to be called by such entry methods as `upload_from_file()` , and the `retry` config and `num_retries` compatible logic is currently handled in `_do_upload()` before calling `_initiate_resumable_upload()`